tests/ensure_provider_tests.py

```diff
@@ -101,10 +101,7 @@
 blackhole, prohibit and unreachable",
     },
     "playbooks/tests_routing_rules.yml": {},
-    # team interface is not supported on Fedora
```

Comment:
Please don't add more commits to this PR that are not needed to fix the Jira issue. It only makes the review more complicated. This can be merged just fine in its own PR.

Reply:
I moved the commit to its own PR.